documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -172,8 +174,10 @@ endif::community[]
 
 [NOTE]
 ====
-To use Kafka signaling to trigger ad hoc incremental snapshots for a connector, you must first xref:debezium-signaling-enabling-source-signaling-channel[enable a `source` signaling channel] in the connector configuration.
+To use Kafka signaling to trigger ad hoc incremental snapshots for most connectors, you must first xref:debezium-signaling-enabling-source-signaling-channel[enable a `source` signaling channel] in the connector configuration.
 The source channel implements a watermarking mechanism to deduplicate events that might be captured by an incremental snapshot and then captured again after streaming resumes.
+Enabling the source channel is not required when using any of others available signaling channels to trigger an incremental snapshot of a read-only MySQL database with {link-prefix}:link-mysql-connector}#enable-mysql-gtids[GTIDs enabled]. 
```

Comment:
```suggestion
Enabling the source channel is not required when using a signaling channel to trigger an incremental snapshot of a read-only MySQL database that has {link-prefix}:link-mysql-connector}#enable-mysql-gtids[GTIDs enabled]. 
```
I thought that this note applied to using the Kafka signaling channel only, but I guess that when using signaling to trigger a snapshot for a MySQL read-only database, you don't ever need to enable the `source` channel?
I've adjusted my suggestion with that in mind.  I think this sets it right, but let me know.  
If you agree with this updated edit, then I think that this change is ready to merge.

Reply:
Before Debezium 2.3 the only way to trigger an incremental snapshot for MySQL with GTIDs and read only mode was Kafka. Now all other channels can be used.